### PR TITLE
Improve Homepage

### PR DESCRIPTION
**Pull request by anshg1214**
This PR is deployed on [https://test.listenbrainz.org/?redirect=false](https://test.listenbrainz.org/?redirect=false)

### CODE REVIEW

**Review by anshg1214 — COMMENTED** [review]
@Aerozol Hi! I've deployed the latest version of the homepage to [test](https://test.listenbrainz.org/?redirect=false). Can you please have a look, and let me know if there is any feedback for the same.

**Comment by MonkeyDo** [comment]
For my screen size, the bottom of the landing page with the text is cropped off, and trying to scroll down snaps back to the top of that section, so the links in each sections are unaccessible:
image


A couple of CSS tweaks to all scrolling to the bottom:

#homepage-container .homepage-information > changing to `height: auto;` and  adding `padding-bottom: 1em;`

File: frontend/js/src/home/Homepage.tsx [hunk 1]
```diff
@@ -248,11 +293,181 @@ function HomePage() {
...
+                alt="Open source"
+              />
+            </div>
+            <h2>Open source</h2>
+            <p>
+              &quot;Open source is source code that is made freely available for
+              possible modification and redistribution... A main principle of
+              open source software development is peer production, with products
+              such as source code, blueprints, and documentation freely
+              available to the public.&quot; - Wikipedia
```
Comment (MonkeyDo) [inline frontend/js/src/home/Homepage.tsx:349]: Have "Wikipedia" link to the actual wiki page

File: frontend/js/src/home/Homepage.tsx [hunk 2]
```diff
@@ -248,11 +293,181 @@ function HomePage() {
...
+            </p>
+            <p>
+              These datasets include the ListenBrainz PostgreSQL Data Dumps,
+              which contains detailed data about ~1 billion listens, and can be
+              used to create and study music consumption patterns. Our datasets
+              are AI Ready, perfect for training large language models for
+              music-based tasks.
+            </p>
+            <p>
+              We ask commercial supporters to support us in order to help fund
```
Comment (MonkeyDo) [inline frontend/js/src/home/Homepage.tsx:407]: Have "commercial supporters" link to this page, maybe?
https://metabrainz.org/supporters/account-type

**Comment by MonkeyDo** [comment]
Looking good !

**Comment by mayhem** [comment]
On my (large) screen the Open Source heading is unreadable:

image

File: frontend/js/src/home/Homepage.tsx [hunk 3]
```diff
@@ -248,11 +293,196 @@ function HomePage() {
...
+              These datasets include the ListenBrainz PostgreSQL Data Dumps,
+              which contains detailed data about ~1 billion listens, and can be
+              used to create and study music consumption patterns. Our datasets
+              are AI Ready, perfect for training large language models for
+              music-based tasks.
+            </p>
+            <p>
+              We ask{" "}
+              <Link
+                to="https://metabrainz.org/supporters/account-type"
```
Comment (MonkeyDo) [inline frontend/js/src/home/Homepage.tsx:416]: Sorry, this page is better, my bad.

```suggestion
                to="https://metabrainz.org/supporters"
```

File: frontend/js/src/home/Homepage.tsx [hunk 4]
```diff
@@ -248,11 +293,196 @@ function HomePage() {
...
+              to further humanity&apos;s global store of knowledge, and to
+              safeguard and share data about the artists and music that we love.
+            </p>
+            <p>
+              Data wants to be free, and we believe that by sharing what we
+              have, rather than jealously guarding it for the benefit of a few,
+              everyone can benefit.
+            </p>
+            <Link
+              to="https://github.com/orgs/metabrainz/repositories"
```
Comment (MonkeyDo) [inline frontend/js/src/home/Homepage.tsx:376]: I think this page is a better overview, with pinned repos and more:
```suggestion
              to="https://github.com/orgs/metabrainz"
```

**Review by MonkeyDo — APPROVED** [review]
Two thumbs up !

**Comment by Aerozol** [comment]
Thanks all! Looks great and super quick work by ansh, as always.
Sorry for the late review - there are some small changes needed/things that don't match the mockup:

1. The three small icons need to disappear on mobile/at very narrow screen widths
2. The three small icons could use some more padding around and between them
3. The lines under the titles (Open source, Data provider, Ethical forever) shouldn't be full width
4. A bit more space between the three columns, as a result it may have to jump to single column sooner

Small changes but I promise they will make a difference

**Comment by Aerozol** [comment]
I've also made some improvements to the images.

The new images are in this folder: https://drive.google.com/drive/folders/1jXy0SuTGNRaco6dbtlQFepWViLX3E5V8?usp=sharing

Should be self-explanatory what they replace. Let me know if you want me to replace them myself :)

**Pull request merged**